### PR TITLE
Added required qualities for a country lead

### PR DESCRIPTION
**Pull request by octaveauger**
@adeclosset take a look, then I'll sent it to Maria for a final check

### CODE REVIEW

**Comment by octaveauger** [comment]
@hirokitakeuchi I updated the job post based on our email chat
@greysteil you can also take a look
Let me know if it's all good for you!

**Comment by greysteil** [comment]
Looks like a good start. @adeclosset - do you want to tag me or @hirokitakeuchi in when you've taken a second cut?

**Comment by adeclosset** [comment]
FYI - @octave-auger it's done. @greysteil tagging Hiroki, but feel free to have a look

@hirokitakeuchi - Want to take the second cut?

File: app/pages/about/_country_lead.html [hunk 1]
```diff
@@ -27,6 +27,22 @@ <h3 class="u-text-heading-light u-text-m u-color-heading">Country Lead: Germany,
   scale in-market growth.
 </p>
 
+<h4 class="u-text-s u-margin-Bxs u-margin-Tm">Strong candidates will demonstrate these qualities:
+</h4>
+<ul class="list">
+  <li>
+    <strong>Self reliant</strong>: you will need to be able to drive in-country work independently and resourcefully, and grow to become the expert of your market
```
Comment (hirokitakeuchi) [inline app/pages/about/_country_lead.html]: Line lengths are too long - you should be starting new lines after 80 characters (grab me tomorrow and I can show you how).

File: app/pages/about/_country_lead.html [hunk 2]
```diff
@@ -27,6 +27,22 @@ <h3 class="u-text-heading-light u-text-m u-color-heading">Country Lead: Germany,
...
+  </li>
+  <li>
+    <strong>Result-oriented</strong>: starting from scratch, you will deliver on launch activities, accompany prospective customers from sales to live, and in general build momentum early on
```
Comment (hirokitakeuchi) [inline app/pages/about/_country_lead.html]: Change "Result-oriented" to "Results oriented"

File: app/pages/about/_country_lead.html [hunk 3]
```diff
@@ -27,6 +27,22 @@ <h3 class="u-text-heading-light u-text-m u-color-heading">Country Lead: Germany,
...
+  </li>
+  <li>
+    <strong>Multifaceted</strong>: you will be responsible for all client-facing activities in-market (marketing, PR, sales, account management, partnerships) and be able to understand and coordinate all product, engineering and support requirements for your market success 
+  </li>
+  <li>
+    <strong>Adaptable</strong>: you will need to be able to communicate complex situations in a clear and simple way to different audiences from different cultures, from the CTO of a prospective client in your local market to the VP of Marketing at GoCardless.
```
Comment (hirokitakeuchi) [inline app/pages/about/_country_lead.html]: This sentence is clunky. Maybe this instead?

you will need to communicate complex situations in a clear and simple way a wide variety of audiences, from CTO's of prospective clients to marketing, engineering, and design teams internally.

**Comment by hirokitakeuchi** [comment]
Broadly looks good. One question - would it make sense to split out each country lead into separate job ads (even if the content of them is duplicated). I feel like it is pretty unclear which countries we are hiring in right now.

**Comment by octaveauger** [comment]
Cheers @hirokitakeuchi, I'll update it accordingly.
I also think splitting it into 3 ads is a good idea for added clarity of
who we are looking for.

**Comment by adeclosset** [comment]
@moyni - Hey Maria, do you want to review and make it consistent with other ads and GC messaging? Thank you!

FYI @octave-auger @hirokitakeuchi - Wording has been updated & job ads split into Germany & Austria / Spain / Netherlands & Belgium. Maria will help to make the ad consistent with others ads & with our brand.

**Comment by adeclosset** [comment]
@moyni - like the changes a lot. I'll quickly change "with lots of travel to the market" to "with travel to the market". "Lots" might give the wrong impression.

**Comment by adeclosset** [comment]
All good on @octaveauger & my side. @hirokitakeuchi - looks good?

**Pull request merged**